tmail-backend/blob/blobid-list/src/main/java/com/linagora/tmail/blob/blobid_list/BlobIdListManager.scala

```diff
@@ -0,0 +1,10 @@
+package com.linagora.tmail.blob.blobid_list
+
+import org.apache.james.blob.api.BlobId
+import org.reactivestreams.Publisher
+
+trait BlobIdListManager {
+  def isStored(blobId: BlobId): Publisher[java.lang.Boolean]
+
+  def store(blobId: BlobId): Publisher[Unit]
```

Comment:
No method to remove stuff?

Reply:
I thought you wrote that "Note deletes in the default bucket should throw" 
https://github.com/linagora/tmail-backend/issues/124
?

Reply:
But hopefully one day there would be deduplication in place and we would be happy to have deletes implemented here already.